src/api/protectedApiClient.ts

```diff
@@ -556,6 +559,12 @@ const sendEmail = (request: SendEmailRequest): Promise<void> => {
   );
 };
 
+const deleteImage = (imageId: number): Promise<void> => {
+  return AppAxiosInstance.delete(
+    ParameterizedApiRoutes.DELETE_IMAGE(imageId),
+  ).then((res) => res.data);
+};
+
```

Comment:
⛏️ Could we add a couple tests for the new route here in `protectedApiClient.test.ts`? These tests are mostly just here as consistency checks to make sure we don't accidentally break a route in the future.